src/P/Function.hs

```diff
@@ -1,12 +1,26 @@
 {-# LANGUAGE NoImplicitPrelude #-}
 module P.Function (
     applyN
+  -- * Functions for development/debuggung only
+  -- | Cannot be used in production code, but might be useful
+  -- during development or debugging
+  , undefined
+  , error
```

Comment:
Yes, I may be better. I was thinking about it but was not sure about the module name. Development make more sense.